components/atoms/DateModified.js

```diff
@@ -11,7 +11,7 @@ export function DateModified(props) {
     <dl className="mt-8 py-2 font-body font-normal text-sm">
       <dt className="inline">{t("dateModified")}</dt>
       <dd className="inline">
-        <time>{` ${dateFormatted}`}</time>
+        <time datetime={dateFormatted}>{` ${dateFormatted}`}</time>
```

Comment:
Unless there is no datetime (eg "NA" would be not right). It's kind of a minor point but it would be good to skip this attribute if `dateFormatted` === `"NA"`

Reply:
Of course good call. Fixing that now.